source/release-notes/backend.txt

```diff
@@ -13,6 +13,35 @@ Atlas App Services Changelog
 2023 Releases
 -------------
 
+.. _backend_20231213:
+
+13 December 2023 Release
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added support for passing in user's new metadata from a request using JWT authentication to the function context.
```

Comment:
```suggestion
- Requests authenticated through the Custom JWT provider now update the user's profile data (e.g. ``context.user.data``) based on the JWT.
```

I think we can focus this less on the implementation and more on its impact to the reader. Also this is kind of a bugfix improvement so maybe we can frame it more like that.